### PR TITLE
[AIRFLOW-XXX] Improvements to formatted content in documentation

### PR DESCRIPTION
**Pull request by jyothsnakanand**
Make sure you have checked _all_ steps below.

### Jira

- [X] My PR fixes typos in the documentation 

### Description
- [X] Here are some details about my PR, including screenshots of any UI changes: 
Formatting changes made in templates with no major changes to core content. 

### Tests

- [X] My PR does not need testing for this extremely good reason: documentation change only

### Commits

- [X] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [X] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
No new functionality in this PR. 

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

File: docs/howto/check-health.rst [hunk 1]
```diff
@@ -33,15 +33,17 @@ To check the health status of your Airflow instance, you can simply access the e
...
+  * The status of ``metadatabase`` depends on whether a valid connection can be initiated with the database
+
+  * The status of ``scheduler`` depends on when the latest scheduler heartbeat was received
+
+    * If the last heartbeat was received more than 30 seconds (default value) earlier than the current time, the scheduler is
+      considered unhealthy
+    * This threshold value can be specified using the option ``scheduler_health_check_threshold`` within the
+      ``scheduler`` section in ``airflow.cfg``
+
+Please keep in mind that the HTTP response code of ``"/health"`` endpoint **should not** be used to determine the health
```
Comment (mik-laj) [inline docs/howto/check-health.rst]:
```suggestion
Please keep in mind that the HTTP response code of ``/health`` endpoint **should not** be used to determine the health
```

Comment (mik-laj) [reply docs/howto/check-health.rst]: The same file has twice this issues.

File: docs/howto/run-with-upstart.rst [hunk 2]
```diff
@@ -22,12 +22,17 @@ Airflow can integrate with upstart based systems. Upstart automatically starts a
...
-You can find sample upstart job files in the ``scripts/upstart`` directory. These files have been tested on
-Ubuntu 14.04 LTS. You may have to adjust ``start on`` and ``stop on`` stanzas to make it work on other upstart
-systems. Some of the possible options are listed in ``scripts/upstart/README``.
+You can find sample upstart job files in the ``scripts/upstart`` directory.
 
-Modify ``*.conf`` files as needed and copy to ``/etc/init`` directory. It is assumed that airflow will run
-under ``airflow:airflow``. Change ``setuid`` and ``setgid`` in ``*.conf`` files if you use other user/group
+The following assumptions have been used while creating these unit files:
+
+1. Airflow will run as the following user:group airflow:airflow
```
Comment (mik-laj) [inline docs/howto/run-with-upstart.rst]:
```suggestion
1. Airflow will run as the following `user:group` ``airflow:airflow``.
```
For consistency with the `run-with-systemd.rst` file

File: docs/howto/set-config.rst [hunk 3]
```diff
@@ -58,9 +58,9 @@ The following config options support this ``_cmd`` version:
...
 
-The order of precedence for all config options is as follows -
+The universal order of precedence for all configuration options is as follows -
 
-1. environment variable
-2. configuration in airflow.cfg
-3. command in airflow.cfg
-4. Airflow's built in defaults
+* set as an environment variable
+* set in airflow.cfg
```
Comment (mik-laj) [inline docs/howto/set-config.rst]: In my opinion, the numbered list makes sense in this case. Shows the hierarchy.

File: docs/lineage.rst [hunk 4]
```diff
@@ -64,10 +64,19 @@ works.
...
 
-Tasks take the parameters `inlets` and `outlets`. Inlets can be manually defined by a list of dataset `{"datasets":
-[dataset1, dataset2]}` or can be configured to look for outlets from upstream tasks `{"task_ids": ["task_id1", "task_id2"]}`
-or can be configured to pick up outlets from direct upstream tasks `{"auto": True}` or a combination of them. Outlets 
-are defined as list of dataset `{"datasets": [dataset1, dataset2]}`. Any fields for the dataset are templated with 
+Tasks take the parameters `inlets` and `outlets`.
+
+Inlets can be manually defined by the following options:
+
+- by a list of dataset `{"datasets": [dataset1, dataset2]}`
```
Comment (mik-laj) [inline docs/lineage.rst]:
```suggestion
- by a list of dataset ``{"datasets": [dataset1, dataset2]}``
```

File: docs/lineage.rst [hunk 5]
```diff
@@ -64,10 +64,19 @@ works.
...
+
+- can be configured to look for outlets from upstream tasks `{"task_ids": ["task_id1", "task_id2"]}`
```
Comment (mik-laj) [inline docs/lineage.rst]:
```suggestion
- can be configured to look for outlets from upstream tasks ``{"task_ids": ["task_id1", "task_id2"]}``
```

File: docs/lineage.rst [hunk 6]
```diff
@@ -64,10 +64,19 @@ works.
...
+
+- can be configured to pick up outlets from direct upstream tasks `{"auto": True}`
```
Comment (mik-laj) [inline docs/lineage.rst]:
```suggestion
- can be configured to pick up outlets from direct upstream tasks ``{"auto": True}``
```

File: docs/lineage.rst [hunk 7]
```diff
@@ -64,10 +64,19 @@ works.
...
+
+- a combination of them
+
+Outlets are defined as list of dataset `{"datasets": [dataset1, dataset2]}`. Any fields for the dataset are templated with
```
Comment (mik-laj) [inline docs/lineage.rst]:
```suggestion
Outlets are defined as list of dataset ``{"datasets": [dataset1, dataset2]}``. Any fields for the dataset are templated with
```

**Comment by mik-laj** [comment]
A good change. Cosmetic changes that are important in large quantities.

I prepared a preview of changes: http://loud-wheel.surge.sh/

**Comment by jyothsnakanand** [comment]
Changes suggested by @mik-laj added to PR. Thanks! 
Commits from the review to be squashed.

File: docs/howto/set-config.rst (hunk 3 above)
Comment (jyothsnakanand) [reply docs/howto/set-config.rst]: sounds good, i'll convert it to a numbered list.

File: docs/howto/check-health.rst (hunk 1 above)
Comment (mik-laj) [reply docs/howto/check-health.rst]: Still not all apostrophes are deleted. Look at the beginning of the file.:-)

Comment (mik-laj) [reply docs/howto/check-health.rst]: I am prepared patch for you
```bash
curl https://pastebin.com/raw/7ijpbhKi | git am
```

Comment (jyothsnakanand) [reply docs/howto/check-health.rst]: my bad, missed the first set of quotes. :)

**Comment by jyothsnakanand** [comment]
@mik-laj , The build stalled on the Kubernetes tests. Does it necessarily have to be fixed with this PR?

**Comment by mik-laj** [comment]
No. It's not required. There are problems with Kubernetes test and we are aware of this. this change does not affect the tests in any way, so you only have to build the documentation correctly. Apache pays for Travis, so if it is not necessary then you should avoid new jobs.

File: docs/howto/check-health.rst [hunk 8]
```diff
@@ -33,15 +33,17 @@ To check the health status of your Airflow instance, you can simply access the e
...
-      with the database backend of Airflow.
-    * The status of ``scheduler`` is depending on when the latest scheduler heartbeat happened. If the latest
-      scheduler heartbeat happened 30 seconds (default value) earlier than the current time, scheduler component is
-      considered unhealthy. You can also specify this threshold value by changing
-      ``scheduler_health_check_threshold`` in ``scheduler`` section of the ``airflow.cfg`` file.
+  * The status of ``metadatabase`` depends on whether a valid connection can be initiated with the database
+
+  * The status of ``scheduler`` depends on when the latest scheduler heartbeat was received
+
+    * If the last heartbeat was received more than 30 seconds (default value) earlier than the current time, the scheduler is
```
Comment (r39132) [inline docs/howto/check-health.rst:42]: There should not be newlines at line 39, 41, 42-43, 46

File: docs/howto/operator/gcp/spanner.rst [hunk 9]
```diff
@@ -272,5 +272,5 @@ Templating
 More information
 """"""""""""""""
 
-See `Google Cloud Spanner API documentation for instance delete
+See `Google Cloud Spanner API documentation to delete an instance
```
Comment (r39132) [inline docs/howto/operator/gcp/spanner.rst]: Change this one as well!

File: docs/howto/operator/gcp/sql.rst [hunk 10]
```diff
@@ -639,5 +640,5 @@ Templating
 More information
 """"""""""""""""
 
-See `Google Cloud SQL Proxy documentation
+See `Google Cloud SQL documentation for more information
```
Comment (r39132) [inline docs/howto/operator/gcp/sql.rst]: Same here

File: docs/tutorial.rst [hunk 11]
```diff
@@ -368,8 +368,10 @@ Testing
 Running the Script
 ''''''''''''''''''
 
-Time to run some tests. First let's make sure that the pipeline
-parses. Let's assume we're saving the code from the previous step in
+Time to run some tests. First, let's make sure the pipeline
+is parsed successfully.
+
```
Comment (r39132) [inline docs/tutorial.rst:373]: Remove this extra newline

File: docs/timezone.rst [hunk 12]
```diff
@@ -34,7 +34,7 @@ happen. (The pendulum and pytz documentation discusses these issues in greater d
 for a simple DAG, but it’s a problem if you are in, for example, financial services where you have end of day
 deadlines to meet.
 
-The time zone is set in `airflow.cfg`. By default it is set to utc, but you change it to use the system’s settings or
+The time zone is set in ``airflow.cfg``. By default it is set to utc, but you change it to use the system’s settings or
```
Comment (r39132) [inline docs/timezone.rst:37]: We need to remove the new line here.

File: docs/lineage.rst [hunk 13]
```diff
@@ -64,10 +64,19 @@ works.
     run_this.set_downstream(run_this_last)
 
 
-Tasks take the parameters `inlets` and `outlets`. Inlets can be manually defined by a list of dataset `{"datasets":
-[dataset1, dataset2]}` or can be configured to look for outlets from upstream tasks `{"task_ids": ["task_id1", "task_id2"]}`
-or can be configured to pick up outlets from direct upstream tasks `{"auto": True}` or a combination of them. Outlets 
-are defined as list of dataset `{"datasets": [dataset1, dataset2]}`. Any fields for the dataset are templated with 
+Tasks take the parameters `inlets` and `outlets`.
+
```
Comment (r39132) [inline docs/lineage.rst:68]: Remove the new lines below

File: docs/kubernetes.rst [hunk 14]
```diff
@@ -23,10 +23,20 @@ Kubernetes Executor
...
 The kubernetes executor is introduced in Apache Airflow 1.10.0. The Kubernetes executor will create a new pod for every task instance.
 
-Example helm charts are available at `scripts/ci/kubernetes/kube/{airflow,volumes,postgres}.yaml` in the source distribution. The volumes are optional and depend on your configuration. There are two volumes available:
+Example helm charts are available at ``scripts/ci/kubernetes/kube/{airflow,volumes,postgres}.yaml`` in the source distribution.
+The volumes are optional and depend on your configuration. There are two volumes available:
 
-- Dags: by storing all the dags onto the persistent disks, all the workers can read the dags from there. Another option is using git-sync, before starting the container, a git pull of the dags repository will be performed and used throughout the lifecycle of the pod.
-- Logs: by storing the logs onto a persistent disk, all the logs will be available for all the workers and the webserver itself. If you don't configure this, the logs will be lost after the worker pods shuts down. Another option is to use S3/GCS/etc to store the logs.
+- **Dags**:
+
```
Comment (r39132) [inline docs/kubernetes.rst:30]: New lines here. Are they needed?

File: docs/howto/write-logs.rst [hunk 15]
```diff
@@ -127,12 +128,12 @@ example:
...
-        [2017-10-03 21:57:51,306] {base_task_runner.py:98} INFO - Subtask: [2017-10-03 21:57:51,306] {models.py:186} INFO - Filling up the DagBag from /airflow/dags/example_dags/example_bash_operator.py
+  *** Reading remote log from gs://<bucket where logs should be persisted>/example_bash_operator/run_this_last/2017-10-03T00:00:00/16.log.
+  [2017-10-03 21:57:50,056] {cli.py:377} INFO - Running on host chrisr-00532
+  [2017-10-03 21:57:50,093] {base_task_runner.py:115} INFO - Running: ['bash', '-c', u'airflow run example_bash_operator run_this_last 2017-10-03T00:00:00 --job_id 47 --raw -sd DAGS_FOLDER/example_dags/example_bash_operator.py']
+  [2017-10-03 21:57:51,264] {base_task_runner.py:98} INFO - Subtask: [2017-10-03 21:57:51,263] {__init__.py:45} INFO - Using executor SequentialExecutor
+  [2017-10-03 21:57:51,306] {base_task_runner.py:98} INFO - Subtask: [2017-10-03 21:57:51,306] {models.py:186} INFO - Filling up the DagBag from /airflow/dags/example_dags/example_bash_operator.py
 
-Note the top line that says it's reading from the remote log file.
+**Note** that the first line that says it's reading from the remote log file.
```
Comment (r39132) [inline docs/howto/write-logs.rst]: Readability ... `that .. that`? Remove the second `that`.

**Review by r39132 — CHANGES_REQUESTED** [review]

File: docs/howto/operator/gcp/sql.rst (hunk 10 above)
Comment (jyothsnakanand) [reply docs/howto/operator/gcp/sql.rst]: adding links for mysql and postgres documentation

File: docs/kubernetes.rst (hunk 14 above)
Comment (jyothsnakanand) [reply docs/kubernetes.rst:30]: newlines required to render bullets as a nested list

**Review by r39132 — APPROVED** [review]

**Pull request merged**